.github/ISSUE_TEMPLATE/prepare_release.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Prepare release
+about: Execute tasks for the creation and publishing of a new release
+title: 'Prepare release 0.0.0'
+labels: release
+assignees: ''
+
+---
+
+<!--
+Add appropriate release number to title!
+
+For detailed info on the release process refer to https://github.com/waku-org/nwaku/blob/master/docs/contributors/release-process.md
+ -->
+ 
+### Items to complete
+- [ ] create release branch
+- [ ] assign release candidate tag
+- [ ] validate release candidate
+- [ ] generate and edit releases notes
+- [ ] open PR and merge release notes to master
+- [ ] cherry-pick release notes to the release branch
+- [ ] assign release tag to the cherry-picked release notes commit
+- [ ] create GitHub release
+- [ ] deploy the release to DockerHub
+- [ ] deploy release to `wakuv2.prod` fleet
+- [ ] announce the release
```

Comment:
That's awesome indeed! :partying_face: !
Additionally, I think it would be outstanding if we could have links to pages where we can have more elaborated information re each step.

Reply:
Yes! I added a comment in the template with the url of the doc with the steps explained in detail

https://github.com/waku-org/nwaku/blob/c65f3e886fca696c94d24883fabf9e8e9c14fcba/.github/ISSUE_TEMPLATE/prepare_release.md?plain=1#L13

How do you think this can be improved?

Reply:
> How do you think this can be improved?

Cool ! I didn't notice that !
IMHO, I think it would be better if we could have a link right after each bullet point, to make it even more explicit

Reply:
Hey @gabrielmer !
Given that we are planning to move all the info into docs.waku.org, I think is better to postpone that detailed link assignation until we have this information there.